code/modules/mob/living/carbon/carbon.dm

```diff
@@ -842,6 +845,9 @@
 			return DEFIB_FAIL_FAILING_HEART
 
 	var/obj/item/organ/brain/current_brain = get_organ_by_type(/obj/item/organ/brain)
+	if(isdullahan(src))
+		var/datum/species/dullahan/dullahan_species = src.dna.species
+		current_brain = locate() in dullahan_species.my_head.loc
```

Comment:
I hate this with every fiber of my being, though my only idea for improving it right now is `/datum/species/proc/get_brain(mob/living/carbon/guy)`